src/components/wallet/methods/createAccount.ts

```diff
@@ -19,6 +19,7 @@ export default async function createAccount(this: Wallet) {
     this.account = {
       publicKey: keypair.publicKey(),
       secretKey: keypair.secret(),
+      availableBalances: false,
```

Comment:
I'm not against having this attribute if its a pain to determine it's value from `ServerApi.ClaimableBalanceRecord`, but it seems to just be a matter of checking an array length.

In which case, I think having an `availableBalances` attribute actually does more harm than good because `availableBalances`'s value and the real `claimableBalanceResp.records.length === 0` might not match if we make a change in future and forget to update `availableBalances`.

Reply:
I'm not a big fan of this implementation tbh but I think I'm going to address this a bit more elegantly in a future PR.

Reply:
Is there a reason we have to keep it as-is for now? It seems to be a easy change to remove the attribute and replaces uses of it with `this.account.claimableBalances.records.length`.

If you have an even better solution I'm curious to hear it